app/i18n/nl/conf.php

```diff
@@ -106,15 +106,15 @@
 			'type' => 'Type',	// IGNORE
 		),
 		'get_all' => 'Toon alle artikelen',
-		'get_category' => 'Toon "%s" categorie',
+		'get_category' => 'Toon “%s” categorie',
```

Comment:
Could you change the rest to „adsf”?

Reply:
Dutch quotation marks are the same as in German?

Reply:
For Dutch everywhere?

Reply:
@math-GH Possibly, but the random German book I grabbed to check uses „adsf“, not „adsf”.

@Alkarex Yes, indeed. :-)

Reply:
Done